docs/src/components/base/borders/borders--2px.html

```diff
@@ -0,0 +1,7 @@
+{% for color in colors %}
+    <div class="swatch h20 mb8">
+      <div class="example swatch-color b2px b-{{ color.colorName }}"></div>
```

Comment:
Again — I think this is acceptable in the world of utility classes. I'm not sure how much we will be using them in production — they're mostly a prototyping tool for the PL at this point, and since you have to type them frequently, they're short and necessarily have some learning curve...

IDK — I look to something like basscss or tachyons, or solid for taxonomy guidance, and they're all pretty succinct.